test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -52,45 +52,138 @@ build_branch()
             ./configure $config && make -j $(grep -c ^processor /proc/cpuinfo)) > /dev/null
 }
 
+#############################################################
+# get_config_file_name:
+#       arg1: branch name
+#############################################################
+get_config_file_name()
+{
+    local file_name=""
+    branch_name=$1
+    if [ "${wt_standalone}" = true ] || [ $older = true ] ; then
+        file_name="${branch_name}/test/format/CONFIG_default"
+        echo $file_name
+        return
+    fi
+    release=$(echo $branch_name |  cut -c9-)
+    file_name="CONFIG_${release}"
+
+    # Special case to generate develop branch CONFIG
+    if [ -z "$release" ] && [ ${branch_name} == "develop" ] ; then
+        release="develop"
+        file_name="CONFIG_${release}"
+    fi
+    echo $file_name
+}
+
+#############################################################
+# create_configs:
+#       arg1: branch name
+#############################################################
+create_configs()
+{
+    branch_name=$1
+
+    file_name=$(get_config_file_name $branch_name)
+
+    if [ -f $file_name ] ; then
+        return
```

Comment:
I'm interested in what measure is expected to be taken if the config file exists. Should we overwrite it and log some message, instead of silently adopt it?

Reply:
Updated the script to print a warning message and overwrite the config file.